monitoring/types.go

```diff
@@ -7,13 +7,18 @@ import (
 const (
 	longhornName = "longhorn"
 
-	subsystemVolume      = "volume"
-	subsystemBackupStats = "backup_stats"
-	subsystemNodeStats   = "node_stats"
+	subsystemVolume          = "volume"
+	subsystemBackupStats     = "backup_stats"
+	subsystemNodeStats       = "node_stats"
+	subsystemInstanceManager = "instance_manager"
+	subsystemManager         = "manager"
 
-	prometheusNodeIDLabel    = "node_id"
-	prometheusVolumeIDLabel  = "volume_id"
-	prometheusConditionLabel = "condition"
+	prometheusNodeIDLabel            = "node_id"
```

Comment:
you can just use NodeLabel ..... InstanceManagerLabel, ManagerLabel, etc

Reply:
Thank you! Fixed